src/pages/RemoveLiquidity/index.tsx

```diff
@@ -219,7 +219,7 @@ export default function RemoveLiquidity({
     if (!currencyAmountA || !currencyAmountB) {
       throw new Error('missing currency amounts')
     }
-    const router = getRouterContract(chainId, library, RoutablePlatform.SWAPR, account)
+    const router = getRouterContract(chainId, library, UniswapV2RoutablePlatform.SWAPR, account)
```

Comment:
Im just wondering for learning purpose this new UniswapV2RoutablePlatform class from SDK was it made to distinguish between regular amm-s and cowswap/curve?